scripts/core/editor3/components/Editor3Component.tsx

```diff
@@ -278,13 +278,22 @@ export class Editor3Component extends React.Component<IProps, IState> {
         }
     }
 
+    isMacOS() {
+        if (navigator.userAgent.indexOf('Mac') != -1) {
```

Comment:
Mac is a very short word and could trigger a false positive when found as a part of some other string https://www.thefreedictionary.com/words-containing-mac

You have it capitalized so it's a bit protected, but I'd rather convert to lower case and check if it includes "macintosh" or "mac os"

Reply:
I agree on your point, but userAgent isn't limitless in regards of what words it might contain. Most answers on the internet say you should check for the string "Mac". Why should I change it if that's the case?